src/messaging/ExchangeMessageDispatch.cpp

```diff
@@ -40,7 +40,7 @@ namespace chip {
 namespace Messaging {
 
 CHIP_ERROR ExchangeMessageDispatch::SendMessage(SecureSessionHandle session, uint16_t exchangeId, bool isInitiator,
-                                                ReliableMessageContext & reliableMessageContext, bool isReliableTransmission,
+                                                ReliableMessageContext * reliableMessageContext, bool isReliableTransmission,
```

Comment:
Are you noticing any memory benefits by changing it to a pointer? Or is this due to some other refactor?

Reply:
It is due to mReliableMessageContext is not available now